CODIGO/Areas/mPooMap.bas

```diff
@@ -258,13 +245,7 @@ Function Map_LegalPos(ByVal X As Integer, ByVal Y As Integer) As Boolean
       
       'Esta el usuario Equitando bajo un techo?
       If UserEquitando And MapData(X, Y).Trigger = eTrigger.BAJOTECHO Or MapData(X, Y).Trigger = eTrigger.CASA Then
-            
-            If Not frmMain.MsgTimeadoOn Then
-                
-                frmMain.MsgTimeadoOn = True
-                frmMain.MsgTimeado = JsonLanguage.item("MENSAJE_MONTURA_SALIR").item("TEXTO")
-            End If
-
+            Call ShowConsoleMsg(JsonLanguage.item("MENSAJE_MONTURA_SALIR").item("TEXTO"))
```

Comment:
`If Not frmMain.MsgTimeadoOn Then` eso esta por que sino cuando queres entrar con una montura a un lugar se spamea la consola de mensajes,